### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by LegendaryMichael**
### What does this change intend to accomplish?

Update to the readme.md:
- Added a description of the difference between AmpliPi and AmpliPro
- Replaced "AmpliPi" with "AmpliPro when in reference to the device
- Added a link to micro-nova.com mentioning we sell the AmpliPro
- Changed two pictures: The front+back photo of the device, and the API diagram

### Checklist

* [x] Have you tested your changes and ensured they work?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-The AmpliPi design is entirely open source, from the software, to the firmware, to the schematics. The REST API and Web App are written in Python and run on a Raspberry Pi 3+ Compute Module. The firmware, which provides the low level volume control and zone configuration, is written in C and configured by the Python API over I2C. All of the internal circuitry comes with full schematics (available in this repo).
+AmpliPro vs AmpliPi: \
+**AmpliPro:** This is the name of the physical hardware device sold by [Micronova](https://www.micro-nova.com/) - the multi-room/zone home audio controller and amplifier unit. \
+**AmpliPi:** This refers to the web app interface that controls the AmpliPro system. AmpliPro hosts a web app (referred to as 'AmpliPi') that can control the system from any PC or mobile phone on the local network.
```
Comment (linknum23) [inline README.md:10]:
```suggestion
**AmpliPi:** This refers to the software and web interface that controls the AmpliPro system. AmpliPro hosts a web app (referred to as 'AmpliPi') that can control the system from any PC or mobile phone on the local network.
```

File: README.md [hunk 2]
```diff
@@ -78,17 +82,17 @@ Here's an example of changing group and zone volumes:
 Check out the full documentation for the web app at [docs/WEB_APP.md](docs/manual/WEB_APP.md).
 
 ## The REST API
-AmpliPi has a REST API that can be used to control volumes, switch and control audio sources, configure different streaming sources, and much more. It allows full configuration and real-time control of the AmpliPi device. The API conforms to the OpenAPI standard. It is fully documented on our [AmpliPi OpenAPI site](https://micro-nova.github.io/AmpliPi).
+AmpliPi has a REST API that can be used to control volumes, switch and control audio sources, configure different streaming sources, and much more. It allows full configuration and real-time control of the AmpliPro device. The API conforms to the OpenAPI standard. It is fully documented on our [AmpliPi OpenAPI site](https://micro-nova.github.io/AmpliPi).
```
Comment (linknum23) [inline README.md:85]:
```suggestion
AmpliPi has a flexible API that can be used to control volumes, switch and control audio sources, configure different streaming sources, and much more. It allows full configuration and real-time control of the AmpliPro device. The API conforms to the OpenAPI standard. It is fully documented on our [AmpliPi OpenAPI site](https://micro-nova.github.io/AmpliPi).
```

File: README.md [hunk 3]
```diff
@@ -78,17 +82,17 @@ Here's an example of changing group and zone volumes:
...
 
 <p align="center">
   <img alt="REST API overview"
-      src="docs/imgs/rest_api_overview.svg">
+      src="docs/imgs/OverviewRestAPI2.png">
   </img>
 </p>
 
-With the REST API, you can easily add automation to your home audio system. Use the API to trigger your AmpliPi system to play music based on smart home events. For example, only play music in zones of your house where motion has been detected, or start playing Pandora when the front door is unlocked.
+With the REST API, you can easily add automation to your home audio system. Use the API to trigger your AmpliPro system to play music based on smart home events. For example, only play music in zones of your house where motion has been detected, or start playing Pandora when the front door is unlocked.
```
Comment (linknum23) [inline README.md:93]:
```suggestion
With its API, you can easily add automation to your home audio system. Use the API to trigger your AmpliPro system to play music based on smart home events. For example, only play music in zones of your house where motion has been detected, or start playing Pandora when the front door is unlocked.
```

**Review by linknum23 — COMMENTED** [review]
While we are updating this I think all uses of "REST API" should be replaced with "API". When I wrote this I didn't really know what I was talking about. REST API is incorrect, our API is not RESTful and does not intend to be.

File: README.md [hunk 4]
```diff
@@ -1,21 +1,25 @@
...
+The AmpliPro design is entirely open source, from the software, to the firmware, to the schematics. The API and Web App are written in Python and run on a Raspberry Pi 3+ Compute Module. The firmware, which provides the low level volume control and zone configuration, is written in C and configured by the Python API over I2C. All of the internal circuitry comes with full schematics (available in this repo).
+
+The system is self-hosted on the Raspberry Pi and is privacy-centric. By design, AmpliPi doesn’t listen to you or spy on you — it just plays your audio! The way it should be. An internet connection is only needed for some external streaming sources, such as Pandora. The Python application running on the Pi hosts a mobile-friendly website and API used to control the system. The website is built on top of the API.
 
 ![High-level Schematic](hw/diagrams/signal_diagram.drawio.svg)
 
 Check us out on [Kickstarter](https://www.kickstarter.com/projects/micro-nova/amplipi-home-audio-system)!
```
Comment (gorski123) [inline README.md:18]: REMOVE! lol

File: README.md [hunk 5]
```diff
@@ -1,21 +1,25 @@
-# AmpliPi Home Audio System
+# AmpliPro Home Audio System
 [![Build Status][workflow-badge]][workflow-link] [![Coverage][coverage-badge]][coverage-link] [![GPL License][license-badge]](COPYING)
```
Comment (gorski123) [inline README.md:2]: Somewhere in this document, we need important links, perhaps right at the top. Links we need are as follows:
1. Online Community (discourse)
2. Latest news and guides (blog)
3. How-to videos (youtube channel)
4. AmpliPro products (www.amplipro.com)

did I miss any other important links?

**Review by gorski123 — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,21 +1,25 @@
...
 
-![AmpliPi](docs/imgs/amplipi.jpg)
+![AmpliPi](docs/imgs/AmpliProOverview2.png)
```
[the remark this answers is not in the document]
Comment (LegendaryMichael) [reply README.md:4]: Do we have an updated photo? I might be able to overlay a screenshot on the laptop in draw.io but the phone will look like a weird rectangle if I try and do the same.

[the remark this answers is not in the document]
Comment (SteveMicroNova) [reply README.md:4]: `/docs/manual/imgs` has the updated version I whipped up for our manual
In my PR (#923) where I create more manuals, that folder structure is rebuilt to be `/docs/manuals/imgs`, whoever is able to merge last will need to keep an eye out to change this to the proper pathing

**Comment by LegendaryMichael** [comment]
@gorski123 @rtertiaer I updated the image and reordered the AmpliPi vs AmpliPro Paragraph. Unless you feel we need to reword anything I'm ready for approval

File: README.md [hunk 7]
```diff
@@ -1,4 +1,7 @@
-# AmpliPro Home Audio System
+# AmpliPi Home Audio Software
```
Comment (linknum23) [inline README.md:1]: I think System is a better description. Software is overly generic and useless its like putting AI in the name of something.

One potential option if system is off the table is Platform but I still prefer System here.

**Comment by linknum23** [comment]
Please use squash and edit the message when merging this. This all should end up as a single commit.

**Comment by SteveMicroNova** [comment]
> Please use squash and edit the message when merging this. This all should end up as a single commit.

a quick guide to squashing, since I feel you might not have come across that yet and it was annoying when I tried to google it way back when because everyone seems to have their own style for some reason:

run 
```
git fetch origin main; git rebase main -i;
```
in your terminal and replace all but the first instance of "pick" with "s" or "squash", then save the file and continue

From there, you'll likely have to run 
```
git push --force
```
to send your squash upstream, forcefully rewriting your history from n commits into a single commit

File: README.md (hunk 7 above)
Comment (LegendaryMichael) [reply README.md]: I changed it to system. I think you're right.

**Comment by LegendaryMichael** [comment]
Changes made, awaiting approval @gorski123

**Review by linknum23 — APPROVED** [review]

**Review by gorski123 — APPROVED** [review]

**Pull request merged**